dm/tests/dmctl_command/run.sh

```diff
@@ -552,6 +566,16 @@ function run_validator_cmd {
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"validation status test" \
 		"new\/ignored\/resolved: 0\/0\/0" 2
+	run_sql_source1 "insert into dmctl_command.t1 values(-10,'ignore-row')"
+	run_sql_source1 "create table dmctl_command.t_trigger_flush110(id int primary key)" # trigger flush
+	sleep 1                                                                             # wait sync
```

Comment:
we can wait syncer `Synced` using query-status, sleep is not stable, below sleep too, if it's stopped we can check it.